test/e2e/service.go

```diff
@@ -47,129 +46,6 @@ var _ = Describe("Services", func() {
 		namespace1 = "e2e-ns-" + dateStamp() + "-1"
 	})
 
-	It("should provide DNS for the cluster", func() {
```

Comment:
The problem with removing it is that we now have no case in the service test that covers UDP, nor a case that covers multi-port services. 

We should at least leave a TODO here saying "TODO: We get coverage of TCP/UDP and multi-port services through the DNS test.  We should have a simpler test for multi-port TCP here, at least.